examples/tti/tti_example.py

```diff
@@ -61,25 +61,24 @@ def setup(dimensions=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
     data.set_receiver_pos(receiver_coords)
     data.set_shape(nt, 101)
 
-    TTI = TTI_cg(model, data, src, t_order=time_order, s_order=space_order, nbpml=nbpml)
-    return TTI
+    return TTI_cg(model, data, src, t_order=time_order, s_order=space_order)
 
 
 def run(dimensions=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
         time_order=2, space_order=4, nbpml=10, dse='advanced', dle='advanced',
-        auto_tuning=False, compiler=None, cache_blocking=None):
-    if auto_tuning:
+        autotune=False, compiler=None, cache_blocking=None):
+    if autotune:
         cache_blocking = None
```

Comment:
It probably is. I tried not to touch TTI any more than I have to in this PR, but I'm happy to clean this up a little further during the rebase.